static/app/types/defaultSearchBar.tsx

```diff
@@ -0,0 +1,21 @@
+import React from 'react';
+import styled from '@emotion/styled';
+
+import space from 'app/styles/space';
+import AsyncView from 'app/views/asyncView';
+
+export type RenderSearch = React.ComponentProps<
+  typeof AsyncView.prototype.renderSearchInput
+>['children'];
+
+export const SearchWrapper = styled('div')`
```

Comment:
Styles don't belong in `static/app/types/`. Can you move this to `static/app/views/settings/components/`?